packages/plugins/src/SidebarPlugin.ts

```diff
@@ -172,29 +152,35 @@ const SidebarPlugin: PluginType<SidebarPluginPage, SidebarPluginOptions, Sidebar
         deep: sidebarRootLevel
       });
 
-      //Map into Sidebar Groups and sort children according to priority, priority = 1 has the highest priority
-      function sortSidebarGroups(sidebarData) {
-        const sortedGroupedPages = sidebarData.map(page => {
-          if (page.childNodes) {
-            const sortedPages = page.childNodes.sort((a, b) => b.priority - a.priority);
-            return { ...page, childNodes: sortedPages };
+      const removeExcludedPages = page => !(page.sidebar && page.sidebar.exclude);
+
+      function sortPagesByPriority(sidebarData) {
+        const pagesByPriority = sidebarData.map(page => {
+          if (page.childNodes.length > 1) {
+            const pagesByPriority = page.childNodes.sort(
+              (pageA, pageB) =>
+                (pageB.priority ? pageB.priority : -1) - (pageA.priority ? pageA.priority : -1)
+            );
+            sortPagesByPriority(page.childNodes);
+            return { ...page, childNodes: pagesByPriority };
           } else {
-            page;
+            return page;
           }
         });
-        return sortedGroupedPages;
+        return pagesByPriority;
       }
 
       await Promise.all(
         rootUserJourneys.map(async dirName => {
           const sidebarFilePath = path.posix.join(String(dirName), options.filename);
-          const pages = await createPageList(dirName);
+          let pages = await createPageList(dirName);
+          pages = pages.filter(page => removeExcludedPages(page));
```

Comment:
FYI... this could of been
          pages = pages.filter(removeExcludedPages);
          but actually a one line function does not have a huge amount of value, so we could inline instead, see other comment
          
          This is a good example of the two possible types of refactoring, in one instance, we could inline and the other we breakout complexity into a named function.